client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -74,7 +76,8 @@ const wooexpress: Flow = {
 		const intent = useSelect( ( select ) => select( ONBOARD_STORE ).getIntent() );
 		const siteSlugParam = useSiteSlugParam();
 
-		const { setStepProgress } = useDispatch( ONBOARD_STORE );
+		const { setStepProgress, setPluginsToVerify } = useDispatch( ONBOARD_STORE );
+		setPluginsToVerify( [ 'woocommerce' ] );
```

Comment:
Not something for this PR (and maybe something we never need), but we may eventually want to have this data keyed by the flow name. I am not sure if we clear the data across flows, but that might be a situation where we could have subtle data pollution or bugs.
```suggestion
		setPluginsToVerify( flowName, [ 'woocommerce' ] );
```